app/assets/javascripts/components/common/campaign_navbar.jsx

```diff
@@ -12,32 +32,25 @@ const CampaignNavbar = ({ campaign }) => {
             <h2 className="title">{I18n.t('campaign.campaign')}: {campaign.title}</h2>
           </a>
           <nav>
-            <div className="nav__item" id="overview-link">
-              <p><a href={`/campaigns/${campaign.slug}/overview`} >{I18n.t('courses.overview')}</a>
-              </p>
+            <div className={`nav__item ${activeTab === 'overview' ? 'active' : ''}`} id="overview-link">
+              <p><a href={`/campaigns/${campaign.slug}/overview`} onClick={() => handleTabClick('overview')}>{I18n.t('courses.overview')}</a></p>
             </div>
-            <div className="nav__item">
-              <p>
-                <a href={`/campaigns/${campaign.slug}/programs`}>{CourseUtils.i18n('courses', campaign.course_string_prefix)}</a>
-              </p>
+            <div className={`nav__item ${activeTab === 'programs' ? 'active' : ''}`}>
+              <p><a href={`/campaigns/${campaign.slug}/programs`} onClick={() => handleTabClick('programs')}>{CourseUtils.i18n('courses', campaign.course_string_prefix)}</a></p>
             </div>
-            <div className="nav__item" id="articles-link">
-              <p>
-                <a href={`/campaigns/${campaign.slug}/articles`}>{I18n.t('courses.articles')}</a>
-              </p>
+            <div className={`nav__item ${activeTab === 'articles' ? 'active' : ''}`} id="articles-link">
+              <p><a href={`/campaigns/${campaign.slug}/articles`} onClick={() => handleTabClick('articles')}>{I18n.t('courses.articles')}</a></p>
             </div>
-            <div className="nav__item">
-              <p>
-                <a href={`/campaigns/${campaign.slug}/users`}>{CourseUtils.i18n('students', campaign.course_string_prefix)}</a>
-              </p>
+            <div className={`nav__item ${activeTab === 'users' ? 'active' : ''}`}>
+              <p><a href={`/campaigns/${campaign.slug}/users`} onClick={() => handleTabClick('users')}>{CourseUtils.i18n('students', campaign.course_string_prefix)}</a></p>
             </div>
-            <div className="nav__item">
-              <p><NavLink to={`/campaigns/${campaign.slug}/ores_plot`}>{I18n.t('courses.ores_plot')}</NavLink></p>
+            <div className={`nav__item ${activeTab === 'ores_plot' ? 'active' : ''}`}>
+              <p><NavLink to={`/campaigns/${campaign.slug}/ores_plot`} onClick={() => handleTabClick('ores_plot')}>{I18n.t('courses.ores_plot')}</NavLink></p>
```

Comment:
Are the changes to the two NavLink entries necessary? Those are the pages that are rendered in React already, so the highlighting currently works.

Reply:
Yes it seems a bit unnecessary, I did it so it would be more uniform but I'll remove that.

Reply:
I think the idea will be that eventually, all of these tabs will be implemented in React, so we want the NavLink examples to be as clean as possible, as that's the pattern I expect the rest of them to follow eventually.